src/routes/docs/advanced/platform/roles/+page.markdoc

```diff
@@ -0,0 +1,287 @@
+---
+layout: article
+title: Roles
+description: Learn how to setup RBAC roles for the Appwrite Console 
+---
+
+The Appwrite Console supports granular permissions to improve team collaboration and security. Each member of your Console team can be assigned a specific role that grants them access to certain areas of your organization's projects. Below is a breakdown of the new roles available, detailing their permissions and intended use cases.
+
+{% info title="Note" %}
+This page covers team member roles for the Appwrite console. Visit the Auth product [roles documentation](https://appwrite.io/docs/products/auth/teams#permissions) if you want to learn more about roles for the [Teams service](https://appwrite.io/docs/references/cloud/client-web/teams).
```

Comment:
I'm worried about saying team in these docs when the Console shows everything as org.

Maybe we can mention org here and then the rest will be team.

```suggestion
This page covers organization member roles for the Appwrite Console. Visit the Auth [roles documentation](https://appwrite.io/docs/products/auth/teams#permissions) if you want to learn more about roles for the [Teams service](https://appwrite.io/docs/references/cloud/client-web/teams).
```